### ISSUE TITLE
Is there a way to disable the default VSC ts error info?

### ISSUE THREAD

**Issue by froggologies** [jliocsar/better-ts-errors#7]
So that it only displays error info from this extension

**Comment by froggologies** [comment]
Never mind, I found this [vscode-disable-ts-errors](https://github.com/zardoy/vscode-disable-ts-errors) extension. Thank you 🙏

**Comment by froggologies** [comment]
Never mind, I found this [vscode-disable-ts-errors](https://github.com/zardoy/vscode-disable-ts-errors) extension. Thank you 🙏

**Comment by froggologies** [comment]
No, it doesn't work. It was working for a moment at the start. But not anymore 🥲

**Comment by jliocsar** [comment]
Hey @froggologies!

I _think_ this extension might be breaking other ones related to error messages after the first time it parses & show errors.

Unfortunately I don't have time to work on this extension lately (tbh coding STALKER mods in Lua has been more fun than this), but I do plan on updating it and re-vamping it to a 2.0 version, fixing a few things and most likely adding options to remove the original error messages and leave only the formatted ones.

I'll leave this issue open to track the feature through it.

Thanks for using this though! Highly appreciated.
